package.json

```diff
@@ -32,10 +32,11 @@
     "@emotion/css": "11.13.5",
     "@grafana/data": "^12.0.0",
     "@grafana/faro-web-sdk": "^1.17.2",
+    "@grafana/i18n": "^12.1.0-249189",
```

Comment:
This is a requirement of the `@grafana/scenes` upgrade, as per [this message](https://raintank-corp.slack.com/archives/C04FRQ2S0AV/p1750937025713649)